packages/scalecube-services/src/services/ServiceCall.js

```diff
@@ -20,37 +23,44 @@ export class ServiceCall {
         this.microservices = ms;
     }
 
-    call(message: Message, type: "Observable" | "Promise") {
```

Comment:
renamed `call` into `initialize` to avoid confuses with `Array.prototype.call` method

Reply:
Agreed on the why but don't like the new name

Maybe service call, it's not init...